src/app/app.routing.ts

```diff
@@ -51,6 +52,7 @@ export const allRoutes: Routes = [
         (c) => c.PublicFormComponent,
       ),
   },
+  { path: "submission-success", component: SubmissionSuccessComponent },
```

Comment:
Could you extract the newly added and the route above (public-form/:id) into a separate Module routes structure, similar to how we do it for AdminModule routes below?

That would nicely encapsulate all related routes into its module.